readme.md

```diff
@@ -1,36 +1,32 @@
-# Quantum Mechanical Keyboard Firmware
+# Vial
 
-[![Current Version](https://img.shields.io/github/tag/qmk/qmk_firmware.svg)](https://github.com/qmk/qmk_firmware/tags)
-[![Discord](https://img.shields.io/discord/440868230475677696.svg)](https://discord.gg/Uq7gcHh)
-[![Docs Status](https://img.shields.io/badge/docs-ready-orange.svg)](https://docs.qmk.fm)
-[![GitHub contributors](https://img.shields.io/github/contributors/qmk/qmk_firmware.svg)](https://github.com/qmk/qmk_firmware/pulse/monthly)
-[![GitHub forks](https://img.shields.io/github/forks/qmk/qmk_firmware.svg?style=social&label=Fork)](https://github.com/qmk/qmk_firmware/)
+[![Docs Status](https://img.shields.io/badge/docs-ready-orange.svg)](https://get.vial.today/)
+[![GitHub contributors](https://img.shields.io/github/contributors/qmk/qmk_firmware.svg)](https://github.com/vial-kb/vial-qmk/pulse/monthly)
+[![GitHub forks](https://img.shields.io/badge/discord-5865F2?link=https%3A%2F%2Fdiscord.com%2Finvite%2FzNKEUXTKwF
+)](https://discord.com/invite/zNKEUXTKwF)
 
-This is a keyboard firmware based on the [tmk\_keyboard firmware](https://github.com/tmk/tmk_keyboard) with some useful features for Atmel AVR and ARM controllers, and more specifically, the [OLKB product line](https://olkb.com), the [ErgoDox EZ](https://ergodox-ez.com) keyboard, and the [Clueboard product line](https://clueboard.co).
+This is a keyboard firmware forked from [QMK](https://github.com/qmk/qmk_firmware) with some useful features such as [Tap Dance](https://get.vial.today/manual/tap-dance.html) and [Combos](https://get.vial.today/manual/combos.html). It allows you to remap your keyboard in real time and addapt your keyboard to your needs. It can be downloaded or used via browser, being a open-source cross platform (Windows, Linux and Mac).
```

Comment:
> This is a keyboard firmware forked from [QMK](https://github.com/qmk/qmk_firmware) with some useful features such as [Tap Dance](https://get.vial.today/manual/tap-dance.html) and [Combos](https://get.vial.today/manual/combos.html). It allows you to remap your keyboard in real time and addapt your keyboard to your needs. It can be downloaded or used via browser, being a open-source cross platform (Windows, Linux and Mac).

This is a keyboard firmware forked from [QMK](https://github.com/qmk/qmk_firmware) with the ability to remap your keyboard in real time and adapt it to your needs through an added on GUI. This can be downloaded as an application or used in a browser, being an open-source cross platform (Windows, Linux and Mac) application. Vial adds several features to QMK's functions, and replaces others with near identical GUI enabled functions. Please refer to the [manual](https://get.vial.today/manual/first-use.html) for more information.